### PR TITLE
Document support in v1.3.2 for large JSON numbers

### PR DESCRIPTION
**Pull request by matthew1001**
See issue https://github.com/hyperledger/firefly/issues/1568

This PR documents the new large-number support. For convenience, here's a preview of the updated topic:

image

### CODE REVIEW

File: doc-site/docs/reference/types/_includes/ffbigint_description.md [hunk 1]
```diff
@@ -7,5 +7,49 @@ strings (with base 10).
 On input you can provide JSON string (string with an `0x` prefix are
 parsed at base 16), or a JSON number.
 
-Be careful when using JSON numbers, that the largest
-number that is safe to transfer using a JSON number is 2^53 - 1.
\ No newline at end of file
+## Maximum size of numbers in versions of FireFly up to `v1.3.1`
```
Comment (EnriqueL8) [inline doc-site/docs/reference/types/_includes/ffbigint_description.md:10]: Looks good but I wouldn't only had this section in the ffbigint description as it's quite nested down in the documentation under reference. We need to document this in two other places:
- Doc around invoking a smart contract through the API
- Release notes for v1.3.2

Comment (EnriqueL8) [reply doc-site/docs/reference/types/_includes/ffbigint_description.md:10]: Thinking something in the interface format https://hyperledger.github.io/firefly/latest/reference/firefly_interface_format/#schema

Comment (EnriqueL8) [reply doc-site/docs/reference/types/_includes/ffbigint_description.md:10]: For example you can see in the simple invoke EVM contract tutorial we use a number https://hyperledger.github.io/firefly/latest/tutorials/custom_contracts/ethereum/#invoke-the-smart-contract

File: doc-site/docs/releasenotes/index.md [hunk 2]
```diff
@@ -4,6 +4,17 @@ title: Release Notes
...
 [Full release notes](https://github.com/hyperledger/firefly/releases)
 
+## [v1.3.2 - Sept 13, 2024](https://github.com/hyperledger/firefly/releases/tag/v1.3.2)
+
+What's New:
+
+- Support for JSON numbers larger than `2^53-1`
+    See [FFBigInt](../reference/types/simpletypes.md#ffbigint) 
+- Ability to install FireFly CLI with Brew for MacOS users
+    See [Brew](../)
```
Comment (EnriqueL8) [inline doc-site/docs/releasenotes/index.md:14]: missing the link here to brew

**Review by EnriqueL8 — CHANGES_REQUESTED** [review]

File: doc-site/docs/releasenotes/index.md (hunk 2 above)
Comment (matthew1001) [reply doc-site/docs/releasenotes/index.md:14]: Bah, my bad.

**Comment by EnriqueL8** [comment]
Will fix codecov in a follow up PR

**Pull request merged**